examples/twittermap/script/reconstructAdmSampleTweetToJson.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
add a usage? and it should be `ToMySQLJson` or `ToPostgreJSON`?

Reply:
It's the same for MySQL or PostgreSQL. The zip file's name might be confusing, so I changed its name from `mysqlSample.json.zip` into `sample.json.zip`.